docs/build/blast-tutorial-2.md

```diff
@@ -16,7 +16,7 @@ Cudos Blast uses Rust Optimiser under the hood to compile contracts ready for de
 First, let's make sure we are at the project root with only one `Cargo.toml` file.
```

Comment:
All directories have at most one `Cargo.toml` file. :)

The root of a Blast project contains a `blast.config.js` file.

Reply:
amended